frontend/src/app/bounty/create/page.tsx

```diff
@@ -57,7 +54,17 @@ const CreateBountyPage: FC = () => {
     const [imgLink, setImgLink] = useState("");
     const [filename, setFilename] = useState<string | undefined>();
 
+    const [minDeadline, setMinDeadline] = useState<Date>(new Date());
+    const blockTimeStamp = useBlockTimestamp();
+    useEffect(() => {
+        if (blockTimeStamp !== undefined) {
```

Comment:
I would add a case where block timestamp is `undefined`.

```suggestion
        if (blockTimestamp === undefined) {
            setMinDeadline(undefined);
        } else {
```

Reply:
Why @guidanoli ? minDeadline is already (after your suggestion) initialized as undefined. 🤔

Reply:
Who knows. 😉 
Done!